docs/reference/eql/detect-threats-with-eql.asciidoc

```diff
@@ -0,0 +1,424 @@
+[role="xpack"]
+[testenv="basic"]
+[[eql-ex-threat-detection]]
+== Example: Detect threats with EQL
+
+experimental::[]
+
+This example tutorial shows you how you can use EQL to detect security threats
+and other suspicious behavior.
+
+In the scenario, you're tasked with detecting
+https://attack.mitre.org/techniques/T1117/[regsvr32 misuse] in Windows event
```

Comment:
The ATT&CK taxonomy has slightly changed since this original PR, so this is the new link. They left in redirects though, so the old one still works, but this is where it redirects to.
```suggestion
https://attack.mitre.org/techniques/T1218/010/[regsvr32 misuse] in Windows event
```